docs/libertyDev.md

```diff
@@ -140,6 +140,6 @@ These can also be specified as command line parameters in addition to the ones i
 | Attribute | Type  | Since | Description | Required |
 | --------- | ----- | ----- | ----------- | -------- |
 | container | boolean | 3.1-M1 (tech preview) | If set to `true`, run the server in the container specified by the `dockerfile` parameter. Setting this to `true` and using the `libertyDev` task is equivalent to using the `libertyDevc` task. The default value is `false` when the `libertyDev` task is used, and `true` when the `libertyDevc` task is used. | No |
-| dockerRunOpts | String | 3.1-M1 (tech preview) | Specifies options to add to the `docker run` command when using dev mode to launch your server in a container. For example, `-e key=value` is recognized by `docker run` to define an environment variable with the name `key` and value `value`. Setting this parameter overrides the `container` parameter to `true`. | No |
-| dockerfile | File | 3.1-M1 (tech preview) | Location of a Dockerfile to be used by dev mode to build the container that runs your server and to specify the context used to build the container image. The default location is `Dockerfile` in the project root. Setting this parameter overrides the `container` parameter to `true`. | No |
+| dockerRunOpts | String | 3.1-M1 (tech preview) | Specifies options to add to the `docker run` command when using dev mode to launch your server in a container. For example, `-e key=value` is recognized by `docker run` to define an environment variable with the name `key` and value `value`. | No |
+| dockerfile | File | 3.1-M1 (tech preview) | Location of a Dockerfile to be used by dev mode to build the container that runs your server and to specify the context used to build the container image. The default location is `Dockerfile` in the project root. | No |
```

Comment:
I know this wasn't a change, but I think we should update the `dockerfile` parameter description:
```
Path to the Dockerfile to be used by dev mode to build the Docker image for the container that will run your Liberty server. The directory containing the Dockerfile will also be the context for the `docker build`. The default path is `Dockerfile` in the project root.
```

@ericglau @turkeylurkey 
thoughts?

Reply:
Sure, but it's not really a path.  If specifying it from the build.gradle, it is a [file](https://docs.gradle.org/current/dsl/org.gradle.api.Project.html#org.gradle.api.Project:file(java.lang.Object)).

Reply:
I like the idea to make the first sentence more exact and to split it into two.